src/main/java/org/kainos/ea/floorIsJavaConfiguration.java

```diff
@@ -2,9 +2,19 @@
 
 import io.dropwizard.Configuration;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import org.hibernate.validator.constraints.*;
+import io.federecio.dropwizard.swagger.SwaggerBundleConfiguration;
 import javax.validation.constraints.*;
+import javax.validation.Valid;
 
 public class floorIsJavaConfiguration extends Configuration {
-    // TODO: implement service configuration
+    @Valid
+    @NotNull
+    private final SwaggerBundleConfiguration swagger = new SwaggerBundleConfiguration();
+
+    @JsonProperty("swagger")
+    public SwaggerBundleConfiguration getSwagger() {
+        swagger.setResourcePackage("org.kainos.ea.resources");
+        String[] schemes = {"http,", "https"};
```

Comment:
don't think that comma should be there
```suggestion
        String[] schemes = {"http", "https"};
```

Reply:
thank you, have resolved!